crypto/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "crypto"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+winterfell = { git = "https://github.com/novifinancial/winterfell"}
+winter_utils = { version = "0.4", package = "winter-utils" }
+rand_utils = { version = "0.4", package = "winter-rand-utils" }
+proptest = "1.0.0"
```

Comment:
Dependencies used for testing purposes only should probably go into `dev-dependencies` section.